spec/system/ai_bot/share_spec.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+RSpec.describe "Share conversation", type: :system, js: true do
```

Comment:
```suggestion
RSpec.describe "Share conversation", type: :system do
```

The `js: true` metadata does not do anything for our system tests setup and is therefore not required.